Cargo.toml

```diff
@@ -78,4 +78,4 @@ It suppots modern features, such as truecolor by default, although ANSI-Colors c
 It also respects enviroment variables, like NO_COLOR, to complety disable colored output. 
 
 For questions, bug reports or feedback, please visit https://github.com/finefindus/artem.
-"""
\ No newline at end of file
+"""
```

Comment:
What did you change here? It looks the same to me.

Reply:
the github editor automatically did this for me